docs/source/tutorials/automated-publication-with-ci.md

```diff
@@ -0,0 +1,160 @@
+
+This tutorial should also work with the "real" Zenodo.
+```
+ 
+## Configure your .gitignore
+ 
+The HERMES workflow (`hermes`) uses temporary caches in `.hermes/`.
+Ignore this directory in your git repository.
+
+Add `.hermes/` to your `.gitignore` file:
+ 
+```{code-block} bash
+:caption: .gitignore
+
+.hermes/
+```
+
+## Provide additional metadata using CITATION.cff
+ 
+To provide high-quality citation metadata for your project and your publication,
+provide a `CITATION.cff` file in the [Citation File Format](https://citation-file-format.github.io/).
+
+If you don't have one yet,
+use the [cffinit](https://citation-file-format.github.io/cff-initializer-javascript/) website
+to create a `CITATION.cff` file.
+Save it to the root directory of your repository, and add it to version control.
+
+```bash
+git add CITATION.cff
+git commit -m "Add citation file"
+git push
+```
+ 
+## HERMES configuration
+ 
+The HERMES workflow is configured in a [TOML](https://toml.io) file: `hermes.toml`.
+Each step in the publication workflow has its own section.
+
+Configure HERMES to:
+
+- harvest metadata from Git and `CITATION.cff`
+- skip validation of `CITATION.cff`
+- deposit on Zenodo Sandbox (which is built on the InvenioRDM)
+- use Zenodo Sandbox as the target publication repository
+
+```{code-block} toml
+:caption: hermes.toml
+:name: hermes.toml
+
+[harvest]
+from = [ "cff", "git" ]
+
+[harvest.cff]
+validate = false
+
+[deposit]
+mapping = "invenio"
+target = "invenio"
+
+[deposit.invenio]
+base_url = "https://sandbox.zenodo.org"
+```
+
+Copy this file to your repository and add it to version control:
+
+```bash
+git add hermes.toml
+git commit -m "Configure HERMES to harvest git and CFF, and deposit on Zenodo Sandbox"
+git push
+```
+
+## Get a personal access token for Zenodo Sandbox
+
+To allow GitHub Actions to publish our repository to Zenodo Sandbox for us,
+we need a personal access token from Zenodo Sandbox.
+
+Log in at https://sandbox.zenodo.org (you may have to register first),
+then [create a personal access token in your user profile](https://sandbox.zenodo.org/account/settings/applications/tokens/new/)
+with the scopes `deposit:actions` and `deposit:write`.
+
+Copy the newly created token into a new [GitHub Secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) called `ZENODO_SANDBOX` in your repository.
+
+## Configure a GitHub Action to automate publication 
+
+The HERMES project provides templates for continous integration systems in a dedicated repository:
+https://github.com/hermes-hmc/ci-templates.
+
+Copy the [template for GitHub to Zenodo Sandbox publication](https://github.com/hermes-hmc/ci-templates/blob/main/TEMPLATE_hermes_github_to_zenodo.yml)
+into the `.github/workflows/` directory in your repository, and rename it as you like (e.g. `hermes_github_to_zenodo.yml`).
+
+Go through the file, and look for comments marked with `# ADAPT`.
```

Comment:
Maybe this is just a litte too lazy. I would exprect at least some synopsis what and why to adapt...

Reply:
Didn't want to overburden the tutorial for this. Once we have a cleaner template (e.g. based on Marketplace Action), we can pull this in here.